src/frontend/src/lib/components/ui/MessageBox.svelte

```diff
@@ -1,25 +1,44 @@
 <script lang="ts">
+	import { slide } from 'svelte/transition';
+	import IconClose from '$lib/components/icons/lucide/IconClose.svelte';
 	import IconInfo from '$lib/components/icons/lucide/IconInfo.svelte';
+	import { SLIDE_EASING } from '$lib/constants/transition.constants';
+	import { i18n } from '$lib/stores/i18n.store';
 
 	export let level: 'plain' | 'info' | 'light-warning' | 'error' = 'info';
+	export let closable = false;
+
+	let closed = false;
```

Comment:
It would be more readable to use an "open" or "visible" (for consistency) state?

(I know existing `InfoBox` is not consistent 😉)

Reply:
ahahahaha sure, going to use `visible`, it seems the most consistent